rpgJavaInterpreter-core/src/test/kotlin/com/smeup/rpgparser/evaluation/Kute10_56.kt

```diff
@@ -1,34 +1,266 @@
-import com.smeup.rpgparser.PerformanceTest
 import com.smeup.rpgparser.interpreter.*
+import com.smeup.rpgparser.parsing.ast.*
+import com.smeup.rpgparser.parsing.ast.AssignmentOperator.*
+import com.strumenta.kolasu.model.ReferenceByName
 import org.junit.Test
-import org.junit.experimental.categories.Category
+import java.nio.charset.Charset
+import kotlin.test.assertEquals
 import kotlin.test.assertTrue
 
+/**
+    Kute10_56.kt is the "hybrid" way to perform the similar MUTE10_56.rpgle DOU (do-until) loop code.
+
+    - (MUTE10_56.rpgle)
+        C                   DOU       $X=10000001
+        C                   EVAL      $X=$X+1
+        C                   ENDDO
+
+    - (Kute10_56.kt)
+        do {
+            execute(statement.body)
+            loopCounter++
+        } while (!eval(statement.endExpression).asBoolean().value)
+
+    "Hybrid" means it uses both Kotlin technology and the Jariko intepreter implementation, so, for exemple
+    the increment of the variable checked for each iteration is done using "EvalStmt" (not simply the plus "+" operator)
+    and the value of the variable is stored (read/write) in globalSymbolTable (SymbolTable) Map.
+    This approach should be useful for developer to be more efficient to develop/debug code due to
+    performance tuning, in a more concise developing environment.
+
+ */
+
 class Kute10_56 {
 
-    var actualElapsedTimeInMillisec = 0L
-    val expectedElapsedTimeInMillisec = 81L
-    var loopCounter = 0L
+    private val charset = Charset.forName("Cp037")
+    private var globalSymbolTable = SymbolTable()
+    private val expectedElapsedTimeInMillisec = 83L
+    private var loopCounter = 0L
 
     @Test
-    @Category(PerformanceTest::class)
     public fun performanceComparing() {
```

Comment:
Removing PerformanceTest category in KUTEnn_mm.kt cause gradle test failing.
Are we sure it's is optimal choice?

Reply:
Removed all Kute's tests. Kute's are used to help in developing.